tests/IResearch/IResearchFilterFunction-test.cpp

```diff
@@ -6343,7 +6409,7 @@ TEST_F(IResearchFilterFunctionTest, levenshteinMatch) {
   // wrong number of arguments
   assertFilterParseFail(
       vocbase(),
-      "FOR d IN myView FILTER LEVENSHTEIN_MATCH(d.foo, 'true', 1, false, 1, 'z') RETURN d");
+      "FOR d IN myView FILTER LEVENSHTEIN_MATCH(d.foo, 'true', 1, false, 1, 'z', 'x') RETURN d");
```

Comment:
Please add a test with invalid parameter type for 'prefix' e,g, array, object or number or null - something like that,